src/components/_classes/component/Component.js

```diff
@@ -2266,7 +2266,7 @@ export default class Component extends Element {
     ], true);
 
     // Lazy load the quill library.
-    return Formio.requireLibrary('quill', 'Quill', _.get(this.options, 'editors.quill.src', `${Formio.cdn.quill}/quill.min.js`), true)
+    return Formio.requireLibrary('quill', 'Quill', _.get(this.options, 'editors.quill.src', `${Formio.cdn.quill}/quill.js`), true)
```

Comment:
Why cannot we use the minified file?
I have checked quill@1.3.7, quill.min.js is half the size of quill.js.